numba/core/callconv.py

```diff
@@ -316,8 +316,9 @@ def get_exception(self, exc_id):
             msg = "unknown error %d in native function" % exc_id
             return SystemError, (msg,)
 
-
-excinfo_t = ir.LiteralStructType([GENERIC_POINTER, int32_t])
+# The structure type constructed by PythonAPI.serialize_uncached()
+# i.e a {i8* pickle_buf, i32 pickle_bufsz, i8* hash_buf}
```

Comment:
The docstring for `serialize_uncached` needs updating, it still refers to the variant without the hash. https://github.com/numba/numba/pull/5922/files#diff-e3116b002a61bdf2a13aff3dda7ed96fR1313

Reply:
updated in https://github.com/numba/numba/pull/5922/commits/ebbae83720e6c29ef6fbb756298211d24869279f